README.md

```diff
@@ -0,0 +1,96 @@
+django-locking
+==============
+
+About
+-----
+
+`django-locking` provides a system that makes concurrent editing impossible, and informs users of what other users are working on and for how long that content will remain locked. Users can still read locked content, but cannot modify or save it.
+
+Requirements
+-------------
+
+* Django 2.2
+* Python 3.6 - 3.8
+
+Installation
+------------
+
+1. Download and install django-locking.
+
+    ```console
+    $ pip install git+git://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Add app to INSTALLED_APPS.
+
+    ```python
+    INSTALLED_APPS = (
+        'locking',
+    )
+    ```
+
+Developing
+----------
+
+1. Clone the git repository:
+
+    ```console
+        git clone https://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Navigate into the cloned repository:
+
+    ```console
+        cd django-locking
+    ```
+
+3. Install the requirements (preferably in a virtual environment):
+
+    ```console
+        python setup.py install
+    ```
+
+4. Run the migrations:
+
+    ```console
+        python manage.py migrate --run-syncdb
+    ```
+5. Collect all static files:
```

Comment:
Why was this added? Generally this is only needed for real deployments, not development, as the built-in Django server knows how to grab the static files when they're in predictable places, as well as it's own provided static files. I tried testing _without_ running this command and it seemed to work fine for me, so I believe we can remove this.

Reply:
When I was reviewing the PR last week, I also tried it without running `collectstatic` and it seemed to be ok. Then I ran it after and it gave a lot of warnings about not copying files that already existed or something like that, but was also ok, so I didn't comment, but also at the time thought about asking if we needed it...

Reply:
ok, I'll try to run it in  a fresh virtual env and remove it if its unncessary. :)

Reply:
It worked for me too in a new virtualenv with a freshly cloned repo. I got rid of `collectstatic` from Readme. Thanks!!